backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -252,7 +261,14 @@ public Response deleteEndpoint(@Context SecurityContext sec, @PathParam("id") UU
                     // Special case wrt OpenBridge
                     if (e.getSubType().equals("slack")) {
                         String processorId = cp.getExtras().get(OB_PROCESSOR_ID);
-                        bridgeApiService.deleteProcessor(bridge.getId(), processorId, bridgeAuth.getToken());
+                        if (processorId != null) { // Should not be null under normal operations.
+                            try {
+                                bridgeApiService.deleteProcessor(bridge.getId(), processorId, bridgeAuth.getToken());
+                            } catch (Exception ex) {
+                                LOGGER.warn("Removal of OB processor failed:" + ex.getMessage());
```

Comment:
```suggestion
                                LOGGER.warn("Removal of OB processor failed", ex);
```

Reply:
I don't think the full trace is needed, message should be enough

Reply:
Same answer as [here](https://github.com/RedHatInsights/notifications-backend/pull/1100#discussion_r836177503).